python/neuroglancer/pipeline/storage.py

```diff
@@ -107,6 +117,50 @@ def get_file(self, file_path):
             content = self._maybe_uncompress(content)
         return content
 
+    def get_files(self, file_paths):
+        """
+        returns a list of files faster by using threads
+        """
+
+        file_paths = deque(file_paths)
+
+        results = []
+
+        def store_result(path, result, error):
+            content, decompress = result
+
+            if content and decompress:
+                content = self._maybe_uncompress(content)
+
+            results.append({
+                "filename": path,
+                "content": content,
+                "error": error,
+            })
+
+        while len(file_paths):
+            path = file_paths.popleft()
+
+            if self._n_threads:
+                callback = partial(store_result, path)
+                self._queue.put(('get_file', callback, path), block=True)
+            else:
+                result = error = None
```

Comment:
It's required because of the loop. If error gets set once (or result fails to get set) the last value will carry over.